test/libs/pipeline_test.py

```diff
@@ -0,0 +1,17 @@
+from libs import pipeline
+from libs.datasets import AggregationLevel
+
+
+def test_location_id_to_level():
+    param_result_map = {
+        "iso1:us#fips:99111": AggregationLevel.COUNTY,
```

Comment:
I don't know if I realized this was a valid location ID - does the county location ID not require state?

Reply:
I created https://trello.com/c/QEbSwjSQ/631-remove-support-for-county-locationid-without-state to fix this.